src/identity.js

```diff
@@ -1830,7 +1830,7 @@ export default function Identity(mpInstance) {
                     userAttributeChanges: {
                         UserAttributeName: key,
                         New: newValue,
-                        Old: previousUserAttributeValue || null,
+                        Old: previousUserAttributeValue,
```

Comment:
Any reason for removing the null here?

Reply:
Another little bug noticed when testing, this is causing the 'old' values in the event batches to be null instead showing their value for the case of falsey attributes. Let me know if this needs to be changed back but isn't the only time we want old to be null when it's never seen before? Hence the condition on line 1822

Reply:
great catch